app/styles/elements/_icon.scss

```diff
@@ -1,3 +1,3 @@
-svg {
+svg:not([fill]) {
```

Comment:
This now overrides some svg's natural colors, it seems. Check the rename button, copy buttons, and icons in the addressBook view.
![Screenshot from 2021-11-11 15-28-01](https://user-images.githubusercontent.com/87635671/141314874-39198fe3-f5f4-4cc9-8c4a-44771448be6f.png)
![Screenshot from 2021-11-11 15-27-21](https://user-images.githubusercontent.com/87635671/141314848-1d9859c8-bf08-461a-93b6-aae4542dc3ce.png)

Reply:
Good catch, will fix!